testfiles/WaterSideEconomizer_PreCoolCoil.rvi

```diff
@@ -0,0 +1,56 @@
+eplusout.eso
+eplusout.csv
+  Site Outdoor Air Drybulb Temperature
+  Zone Mean Air Humidity Ratio
+  Zone Air Relative Humidity
+  Zone Air Temperature
+  System Node Temperature
+  System Node Mass Flow Rate
+  Zone Predicted Sensible Load to Setpoint Heat Transfer Rate
+  Zone Predicted Moisture Load Moisture Transfer Rate
+  Zone Air System Sensible Cooling Rate
+  Zone Air System Sensible Heating Rate
+  Cooling Coil Air Mass Flow Rate
+  Cooling Coil Air Inlet Temperature
+  Cooling Coil Air Inlet Humidity Ratio
+  Cooling Coil Latent Cooling Rate
+  Cooling Coil Air Outlet Temperature
+  Cooling Coil Air Outlet Humidity Ratio
+  Cooling Coil Sensible Cooling Rate
+  Cooling Coil Total Cooling Rate
+  Cooling Coil Part Load Ratio
+  Cooling Coil Electricity Rate
+  Cooling Coil Runtime Fraction
+  Fan Electricity Rate
+  Heating Coil Heating Rate
+  Heating Coil NaturalGas Rate
+  Zone Air Terminal VAV Damper Position
+  Zone Air Terminal Minimum Air Flow Fraction
+  Zone Air Terminal Outdoor Air Volume Flow Rate
+  Mixed Air Node,System Node Temperature
+  FreeClgCoil OutletNode,System Node Temperature
+  Air System Outdoor Air Flow Fraction
+  Air System Outdoor Air Mass Flow Rate
+  Air System Mixed Air Mass Flow Rate
+  VAV Sys 1 Outlet Node,System Node Temperature
+  VAV Sys 1 Outlet Node,System Node Mass Flow Rate
+  VAV Sys 1 Outlet Node,System Node Setpoint Temperature
+  VAV Sys 1 Outlet Node,System Node Humidity Ratio
+  Unitary System Part Load Ratio
+  Unitary System Total Cooling Rate
+  Unitary System Sensible Cooling Rate
+  Unitary System Latent Cooling Rate
+  Unitary System Total Heating Rate
+  Unitary System Sensible Heating Rate
+  Unitary System Latent Heating Rate
+  Unitary System Ancillary Electricity Rate
+  Unitary System Dehumidification Induced Heating Demand Rate
+  Unitary System Compressor Part Load Ratio
+  Unitary System Frost Control Status
+  System Node Current Density Volume Flow Rate
+  Coil System Water Part Load Ratio
+  Coil System Water Total Cooling Rate
+  Coil System Water Sensible Cooling Rate
+  Coil System Water Latent Cooling Rate
+  Coil System Water Control Status
+0
```

Comment:
Add this rvi file to ensure all outputs are the same, due to output variable order changes.